tests/conftest.py

```diff
@@ -1,6 +1,11 @@
 # conftest.py is a special pytest file. Fixtures defined in this file will be accessible to all tests in this directory
 # without needing to explicitly import them. (https://docs.pytest.org/en/6.2.x/fixture.html)
 
+
 # We import the model_installer and torch_device fixtures here so that they can be used by all tests. Flake8 does not
 # play well with fixtures (F401 and F811), so this is cleaner than importing in all files that use these fixtures.
 from invokeai.backend.util.test_utils import model_installer, torch_device  # noqa: F401
+
+pytest_plugins = [
+    "tests.fixtures.sqlite_database",
```

Comment:
The `pytest` docs suggested that this is the preferred way to declare global fixtures, but I may have misunderstood this, since the tests continue to work after deleting this section. So I'll be removing.